test/unit/org/apache/cassandra/io/sstable/LegacySSTableTest.java

```diff
@@ -94,7 +94,7 @@
      * See {@link #testGenerateSstables()} to generate sstables.
      * Take care on commit as you need to add the sstable files using {@code git add -f}
      */
-    public static final String[] legacyVersions = {"nb", "na", "me", "md", "mc", "mb", "ma"};
+    public static final String[] legacyVersions = {"oa", "nb", "na", "me", "md", "mc", "mb", "ma"};
```

Comment:
Isn't `oa` the current version?

Reply:
Yes but it needs to be added here. `When adding a new sstable version, add that one here` just a few lines above. Also notice how `nb` was there when it was the current version.

Reply:
Makes sense, it's the `legacyVersions` name what confused me.

Reply:
In case you missed my comment in Jira, this patch should not introduce a new major sstable version.

Reply:
I saw it, but I go one step at a time.

Reply:
Started a conversation [here](https://lists.apache.org/thread/1skggvfqzsznrrq0o5ztc58oxowh1x0g) to gather details.